diff --git a/modules/user/user.module b/modules/user/user.module
index 60f32a1..8e26b1d 100644
--- a/modules/user/user.module
+++ b/modules/user/user.module
@@ -4023,7 +4023,7 @@ function user_file_download_access($field, $entity_type, $entity) {
  * @todo: Remove in D8, pending upgrade path.
  */
 function user_system_info_alter(&$info, $file, $type) {
-  if ($type == 'module' && $file->name == 'profile' && db_table_exists('profile_field')) {
+  if ($type == 'module' && $file->name == 'profile' && function_exists('db_table_exists') && db_table_exists('profile_field')) {
     $info['hidden'] = FALSE;
   }
 }
